src/rules/comment-empty-line-before/__tests__/index.js

```diff
@@ -17,12 +17,16 @@ function alwaysTests(tr) {
   tr.ok("a {}\r\n\r\n/** comment */", "CRLF")
   tr.ok("a { color: pink;\n\n/** comment */\ntop: 0; }")
 
-  tr.notOk("/** comment */\n/** comment */", messages.expected)
-  tr.notOk("/** comment */\r\n/** comment */", messages.expected, "CRLF")
   tr.notOk("a { color: pink;\n/** comment */\ntop: 0; }", messages.expected)
   tr.notOk("a { color: pink;\r\n/** comment */\r\ntop: 0; }", messages.expected, "CRLF")
 }
 
+function alwaysTests(tr) {
+  alwaysTestsCommon(tr)
+  tr.notOk("/** comment */\n/** comment */", messages.expected)
+  tr.notOk("/** comment */\r\n/** comment */", messages.expected, "CRLF")
+}
+
 testRule("always", tr => {
   alwaysTests(tr)
 
```

Comment:
`alwaysTests` is used in 3 different places, and that is why I wrapped the tests like that. Are you sure that you want me to copy/paste the two tests inside `alwaysTests` into 3 different locations?